docs/source/pyfstat.rst

```diff
@@ -1,11 +1,6 @@
 pyfstat package
 ===============
 
-These pages document the full API and set of classes provided by PyFstat.
-
-See :doc:`here <readme>` for installation instructions and
-other general information.
-
```

Comment:
please reinstate

Reply:
...but those sentences still show up at https://pyfstat--379.org.readthedocs.build/en/379/modules.html so I'm confused...

Reply:
...ah, the output page where they've disappeared is https://pyfstat--379.org.readthedocs.build/en/379/pyfstat.html